Detections/SecurityEvent/NonDCActiveDirectoryReplication.yaml

```diff
@@ -0,0 +1,56 @@
+id: b9d2eebc-5dcb-4888-8165-900db44443ab
+name: Non Domain Controller Active Directory Replication
+description: |
+  'This query uses Windows Event ID 4662 in order to detect potential attempts to synz an active directory object leveraging directory replication services (DRS)
+  from a non-domain-controller account with privileged permissions to do so. Ref: https://threathunterplaybook.com/notebooks/windows/06_credential_access/WIN-180815210510.html'
+severity: High
+requiredDataConnectors:
+  - connectorId: SecurityEvents
+    dataTypes:
+      - SecurityEvents
+queryFrequency: 1d
+queryPeriod: 7d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Credential Access
+relevantTechniques:
+  - T1003
+query: |
+  let timeframe = 7d;
+  // Enter a reference list of hostnames for your DC servers
+  let DCServersList = dynamic (["DC01.simulandlabs.com"]);
+  SecurityEvent
+  | where TimeGenerated > ago(timeframe)
+  | where Computer in (DCServersList)
+  | where EventID == 4662 and ObjectServer == 'DS'
+  | where AccountType != 'Machine'
+  | where Properties has '1131f6aa-9c07-11d1-f79f-00c04fc2dcd2' //DS-Replication-Get-Changes
+      or Properties has '1131f6ad-9c07-11d1-f79f-00c04fc2dcd2' //DS-Replication-Get-Changes-All
+      or Properties has '89e95b76-444d-4c62-991a-0facbeda640c' //DS-Replication-Get-Changes-In-Filtered-Set
+  | project TimeGenerated, Account, Activity, Properties, SubjectLogonId, Computer
+  | join kind=inner
+  (
+      SecurityEvent
+      | where TimeGenerated > ago(timeframe)
+      | where Computer in (DCServersList)
+      | where EventID == 4624 and LogonType == 3
```

Comment:
Should we assume Logon Type 3 here?  Is there potential for other types outside of Network, such as Interactive, Batch or Service?

Reply:
Only Network Logon type applies here since the replication request is performed from a remote endpoint

Reply:
fixed